index.html

```diff
@@ -0,0 +1,983 @@
+                    </div>
+                    <div class="col-md-3 right">
+                        <a class="subscribe">
+                            <span>SUBSCRIBE</span> 
+                            <svg version="1.1" id="Layer_2" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 8 14" style="enable-background:new 0 0 8 14;" xml:space="preserve"><g id="arrow-right"><defs><rect id="SVGID_1_" width="8" height="14"></rect></defs><clipPath id="SVGID_2_"><use xlink:href="#SVGID_1_" style="overflow:visible;"></use></clipPath><g id="Shape_1" class="st0"><g><polygon class="st1" points="8,7 1.1,0 0,1.1 5.9,7 0,12.9 1.1,14 8,7 8,7	"></polygon></g></g></g></svg>
+                        </a>                    
+                        <a class="signIn">SIGN IN</a>
+    
+                    </div>
+                </div>
+                <div class="row navbarBottom">
+                    <div class="topRightSpan">
+                        <span></span>
+                    </div>
+                    <nav class="navbar navbar-light bg-light navbarBellowRedtrack">
+                        <ul class="navbar-nav">
+                            <li class="nav-item active">
+                                <a class="nav-link" href="#">U.S.<span class="sr-only">(current)</span></a>
+                            </li>
+                            <li class="nav-item">
+                                <a class="nav-link" href="#">World</a>
+                            </li>
+                            <li class="nav-item">
+                                <a class="nav-link" href="#">Business</a>
+                            </li>
+                            <li class="nav-item">
+                                <a class="nav-link" href="#">Tech & Science</a>
+                            </li>
+                            <li class="nav-item">
+                                <a class="nav-link" href="#">Culture</a>
+                            </li>
+                            <li class="nav-item">
+                                <a class="nav-link" href="#">Sports</a>
+                            </li>
+                            <li class="nav-item">
+                                <a class="nav-link" href="#">Health</a>
+                            </li>
+                            <li class="nav-item">
+                                <a class="nav-link imageLink" href="#"><img src="images/NDELIBLE.black3.gif" alt=""></a>
+                            </li>                                                                                                    
+                        </ul>
+                        <form class="form-inline">
```

Comment:
I've been adding comments about the possibility to have an input without a form. Haven't found any reference to a "best practice". It depends on what do you want to achieve. But I just want to emphasize that it is perfectly possible and viable to have an input without a form

Reply:
Thanks, you have right but in this case form have it"s own style... In the real case it could be real form.